Tests/IntegrationTestBundle/Resources/config/form.xml

```diff
@@ -11,17 +11,17 @@
 	<services>
 		<service id="integrationTestBundle.form.createTopic"
 				class="Craue\FormFlowBundle\Tests\IntegrationTestBundle\Form\CreateTopicForm">
-			<tag name="form.type" alias="createTopic" />
+			<tag name="form.type" />
 		</service>
 
 		<service id="integrationTestBundle.form.createVehicle"
 				class="Craue\FormFlowBundle\Tests\IntegrationTestBundle\Form\CreateVehicleForm">
-			<tag name="form.type" alias="createVehicle" />
+			<tag name="form.type" />
 		</service>
 
 		<service id="integrationTestBundle.form.issue64"
 				class="Craue\FormFlowBundle\Tests\IntegrationTestBundle\Form\Issue64Form">
-			<tag name="form.type" alias="issue64" />

```

Comment:
yeah, exactly (well, 2.8 will register the legacy names too, but it will trigger a deprecation warning only if you *use* type through the legacy name, not if you just make it available)